src/primitives/transaction.h

```diff
@@ -501,9 +509,12 @@ class CTxIn
 class CTxInWitness
 {
 public:
+    // TODO generalize CScriptWitness into just CWitness
     std::vector<unsigned char> vchIssuanceAmountRangeproof;
     std::vector<unsigned char> vchInflationKeysRangeproof;
     CScriptWitness scriptWitness;
+    // Re-use script witness struct to include its own witness
+    CScriptWitness pegin_witness;
```

Comment:
If you're going to follow the bitcoin style guides, this should be `m_pegin_witness`.